packages/core/src/features/st-var.ts

```diff
@@ -286,17 +325,14 @@ function evaluateValueCall(
                     args: restArgs,
                     node: resolvedVarSymbol.node,
                     meta: resolvedVar.meta,
+                    rootArgument: varName,
+                    evaluatorNode: node,
```

Comment:
`evaluatorNode` is probably a more fitting name is `initialNode`.
In future refactor it should on the context (currently add a "TODO" comment)